lib/block_view.go

```diff
@@ -8048,6 +8257,65 @@ func (bav *UtxoView) ConnectBlock(
 	return utxoOps, nil
 }
 
+func (bav *UtxoView) Preload(bitcloutBlock *MsgBitCloutBlock) error {
+	// We can only preload if we're using postgres
+	if bav.Postgres == nil {
+		return nil
+	}
+
+	// One iteration for all the PKIDs
+	var publicKeys []*PublicKey
+	for _, txn := range bitcloutBlock.Txns {
+		if txn.TxnMeta.GetTxnType() == TxnTypeFollow {
```

Comment:
Why only follow? I assume because it was slow for some reason? Again, comments here would be helpful...

Reply:
This is WIP. Follows are one of the slower parts of the sync.